src/utils/avatar-volume-utils.js

```diff
@@ -0,0 +1,56 @@
+export const MAX_VOLUME_LABELS = 21;
```

Comment:
I think most of the complexity in this file goes away if useAvatarVolume just operates only in gainMultiplier. The one spot that actually cares about level (for displaying the |||| volume bar) can just keep that logic inlined since its purely display logic.

Reply:
updated